app/recurring_job.go

```diff
@@ -142,51 +145,62 @@ func recurringJob(c *cli.Context) error {
 	logger.Infof("Found %v volumes with recurring job %v", len(filteredVolumes), jobName)
 
 	concurrentLimiter := make(chan struct{}, jobConcurrent)
-	var wg sync.WaitGroup
-	defer wg.Wait()
+	ewg := &errgroup.Group{}
+	defer func() {
+		if wgError := ewg.Wait(); wgError != nil {
```

Comment:
note: Wait blocks until all function calls from the Go method have returned, then returns the first non-nil error (if any) from them.

ref: https://pkg.go.dev/golang.org/x/sync/errgroup#Group.Wait